### PR TITLE
tables access internal

### PR DESCRIPTION
**Pull request by rvilarl**
@ronyeh as you suggested CommonMetrics is now on a different file as Metrics. I have also provided access to UNISON in flow to avoid the usage of Tables in the tests.

No visual differences.

### CODE REVIEW

File: src/accidental.ts [hunk 1]
```diff
@@ -61,10 +62,10 @@ export class Accidental extends Modifier {
     // If there are no accidentals, no need to format their positions.
     if (!accidentals || accidentals.length === 0) return;
 
-    const noteheadAccidentalPadding = Tables.lookupMetric('Accidental.noteheadAccidentalPadding');
+    const noteheadAccidentalPadding = Metrics.lookupMetric('Accidental.noteheadAccidentalPadding');
```
Comment (ronyeh) [inline src/accidental.ts:65]: Can we rename to the shorter version:

`Metrics.lookup(...)`

Comment (ronyeh) [reply src/accidental.ts:65]: Or... how about

```
Metrics.get(...)
Metrics.set(...)
```

It's short and easy to understand. If we don't want to implement set for now, that's fine. But I can see a situation where a developer might want to temporarily change the metrics for a certain group of music symbols.

File: src/flow.ts [hunk 2]
```diff
@@ -241,6 +242,14 @@ export class Flow {
...
   }
 
+  static get UNISON(): boolean {
+    return Tables.UNISON;
+  }
+
+  static set UNISON(unison: boolean) {
+    Tables.UNISON = unison;
+  }
+
```
Comment (ronyeh) [inline src/flow.ts:252]: Interesting. What does this flag do? It is only referenced on one line in the repo. Maybe in a different PR we can figure out which class should store this flag? Is it Metrics related, or is it StaveNote related?

File: src/metrics.ts [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+      if (keyPart) {
+        curr = curr[keyPart]; // Go down one level.
+      } else {
+        break;
+      }
+    }
+
+    return retVal;
+  }
+}
```
Comment (ronyeh) [inline src/metrics.ts:51]: Should there be an exra new line in here? Just checking to see if eslint would recommend the empty line.

File: src/tables.ts [hunk 4]
```diff
@@ -555,52 +555,6 @@ export class Tables {
     return clefs[clef];
   }
 
-  /** Use the provided key to look up a FontInfo in CommonMetrics. **/
-  static lookupMetricFontInfo(key: string): Required<FontInfo> {
```
Comment (ronyeh) [inline src/tables.ts:559]: Do you also plan to delete the big `CommonMetrics` from tables.ts?

File: src/flow.ts (hunk 2 above)
Comment (ronyeh) [reply src/flow.ts:252]: I guess Flow is fine for now....

**Review by ronyeh — CHANGES_REQUESTED** [review]
Thanks!

File: src/tables.ts (hunk 4 above)
Comment (rvilarl) [reply src/tables.ts:559]: That was the aim, and I forgot :). Done

File: src/metrics.ts (hunk 3 above)
Comment (rvilarl) [reply src/metrics.ts:51]: done

File: src/flow.ts (hunk 2 above)
Comment (rvilarl) [reply src/flow.ts:252]: Left unmodified

File: src/accidental.ts (hunk 1 above)
Comment (rvilarl) [reply src/accidental.ts:65]: Now MetricsDefaults is accesible, but a set would be easier. I have no idea how to code that. Could you do it on another PR?

Comment (rvilarl) [reply src/accidental.ts:65]: lookup changed to get.

**Pull request merged**